### ISSUE TITLE
IdsCalendar: Support for displayRange

### ISSUE THREAD

**Issue by JunyiZhao720** [infor-design/enterprise-wc#1468]
**Is your feature request related to a problem or use case? Please describe.**
We need a feature for IdsCalendar to support date range input. When we set it like this: 
`<ids-calendar #calendar [startDate] = "07/03/2022" [endDate] = "08/15/2022", ></ids-calendar>`
The calendar should render strictly from 07/03/2022 to 08/15/2022.

**Describe the solution you'd like**
Currently, we can use a subcomponent of IdsCalendar to set the date range:
`const view = this.calendar.nativeElement.getView() as IdsMonthView

view.startDate = "07/18/2021";
view.endDate = "02/03/2022"`

However, with this approach:
1. The date picker of IdsCalendar won't refresh immediately. For example, if the date picker is set to "September 2023" and we change the date range using monthview, it won't automatically refresh.
2. The first day of the month is missing.
![image](https://github.com/infor-design/enterprise-wc/assets/22669426/29f2a48b-c768-416a-ac6e-cb342ed73b5c)
As you can see, the first day of any month should be "1", but on the desktop, after we change the date range using monthview, it becomes empty. 
For the mobile view, the first day of the month is represented by its letters rather than "1".
![image](https://github.com/infor-design/enterprise-wc/assets/22669426/55ccb094-1cbd-4e87-98f6-8d0597e67791)

The ideal implementation might involve changing the subcomponent's date range, but other components should sync with any date range changes.

**Comment by JunyiZhao720** [comment]
Yes. Some of our features will have the month view starting and ending within a month. This means that the start day is not necessarily the first day of the month, and the same goes for the end date. Additionally, we might display more than one month's view in a span shorter than one month, so it's not strictly a one-month view.

**Comment by tmcconechy** [comment]
But its basically one month with a bit more days? Not even half a month? And not 2/3/4 months? Asking because that would make this easier i think.

**Comment by tmcconechy** [comment]
OK i still think this is a (5) request. Given all the variables. By paging you mean the next and previous button?

**Comment by JunyiZhao720** [comment]
yeah. one of our feature would have next/prev button to switch calendar view back and forth in a long range.
